tabs/system-setup/system-cleanup.sh

```diff
@@ -0,0 +1,64 @@
+#!/bin/sh -e
+
+. ../common-script.sh
+
+cleanup_system() {
+    printf "%b\n" "${YELLOW}Performing system cleanup...${RC}"
+    case "$PACKAGER" in
+        apt-get|nala)
+            $ESCALATION_TOOL "$PACKAGER" clean
+            $ESCALATION_TOOL "$PACKAGER" autoremove -y
+            $ESCALATION_TOOL "$PACKAGER" autoclean
+            $ESCALATION_TOOL du -h /var/cache/apt
+            $ESCALATION_TOOL "$PACKAGER" clean
+            ;;
+        zypper)
+            $ESCALATION_TOOL "$PACKAGER" clean -a
+            $ESCALATION_TOOL "$PACKAGER" tidy
+            $ESCALATION_TOOL "$PACKAGER" cc -a
+            ;;
+        dnf)
+            $ESCALATION_TOOL "$PACKAGER" clean all
+            $ESCALATION_TOOL "$PACKAGER" autoremove -y
+            ;;
+        pacman)
+            $ESCALATION_TOOL "$PACKAGER" -Sc --noconfirm
+            $ESCALATION_TOOL "$PACKAGER" -Rns "$(pacman -Qtdq)" --noconfirm
+            ;;
+        *)
+            printf "%b\n" "${RED}Unsupported package manager: $PACKAGER${RC}"
+            return 1
+            ;;
+    esac
+
+    printf "%b\n" "${GREEN}System cleanup completed.${RC}"
```

Comment:
Remove this line or change it to
```suggestion
    printf "%b\n" "${GREEN}Package manager cleanup completed.${RC}"
```